### PR TITLE
docs: add 1.13 release notes

### LINKED ISSUE #9212: Podman Desktop 1.13 Release Notes
### Is your enhancement related to a problem? Please describe

Write the release note for the Podman Desktop 1.13 release

### Describe the solution you'd like

Release note to align with Podman desktop 1.13 release

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by SoniaSandler**
Signed-off-by: Sonia Sandler <ssandler@redhat.com>

### What does this PR do?
Adds the new release notes for PD 1.13 version

### What issues does this PR fix or reference?
Closes https://github.com/containers/podman-desktop/issues/9212

### How to test this PR?

- [ ] Tests are covering the bug fix or the new feature

### CODE REVIEW

File: website/blog/authors.yml [hunk 1]
```diff
@@ -56,4 +56,10 @@ cedric:
   name: Cedric Clyburn
   title: Developer Advocate
   url: https://github.com/cedricclyburn
-  image_url: https://github.com/cedricclyburn.png
\ No newline at end of file
+  image_url: https://github.com/cedricclyburn.png
+
+SoniaSandler:
+  name: Sonia Sandler
+  title: Associate Software Engineer
```
Comment (deboer-tim) [inline website/blog/authors.yml:63]: You don't have to put your official job title here, could go with 'developer' or whatever you want. :)

File: website/blog/2024-10-07-release-1.13.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+Podman Desktop 1.13 Release! 🎉
+
+![podman-desktop-hero-1.13](img/podman-desktop-release-1.13/banner.png)
+
+Podman Desktop 1.13 is now available. [Click here to download it](/downloads)!
+
+This release includes:
+
+- **HyperV support**: You can now create and manage HyperV Podman machines directly from Podman Desktop.
+- **Search image feature**: search for an image from Podman Desktop.
```
Comment (deboer-tim) [inline website/blog/2024-10-07-release-1.13.md]: `search` (and `start` on the next line) should be capitalized.

File: website/blog/2024-10-07-release-1.13.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+- **New Kubernetes navigation**: All Kuberenetes related pages have been moved to a separate sub menu for easier and less crowded navigation.
+
+<!--truncate-->
+
+## Release Details
+
+### HyperV support
+
+Podman Desktop is now supporting the creation and modification of HyperV based Podman machines in Windows directly from the app.
+When creating a podman machine in Windows, if both HyperV and WSL are enabled, you can choose either one of them as the provider from the dropdown menu.
```
Comment (deboer-tim) [inline website/blog/2024-10-07-release-1.13.md]: This isn't referring to the cli command, so Podman should be capitalized.

File: website/blog/2024-10-07-release-1.13.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+
+### Search image feature
+
+Want to pull an image but forgot its full name or tag? Now when pulling an image, a list of possible images is shown based on the provided input to make it easier for your to find and pull the right image. Just enter an image name and look for the right one in the list. If you would like to search for a specific tag, just add : to the name.
```
Comment (deboer-tim) [inline website/blog/2024-10-07-release-1.13.md]: `Now when pulling an image, a list of possible images is shown based on the provided input to make it easier for your to find and pull the right image. Just enter an image name and look for the right one in the list.` is a bit of a mouthful.
How about just `As you type a list of likely images is now shown to help you find the right one and reduce typing` along with a small image?

File: website/blog/2024-10-07-release-1.13.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+
+### Updated empty state pages
+
+We have updated the empty state pages for images, containers, pods, and Kubernetes. You can now pull your first image, create your first container or pod, all with only one click.
+
+![images empty page](img/podman-desktop-release-1.13/images_empty_page.png)
+
+### New Kubernetes navigation
+
+As we grow our Kubernetes section and add new Kubernetes related pages, its navigation could become a bit crowded. For better user exprience, this release includes a new Kubernetes sub menu with all the relevant pages in it.
```
Comment (deboer-tim) [inline website/blog/2024-10-07-release-1.13.md]: 'its navigation could become' -> 'the navigation was becoming'?

File: website/blog/2024-10-07-release-1.13.md [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+
+### Updated empty state pages
+
+We have updated the empty state pages for images, containers, pods, and Kubernetes. You can now pull your first image, create your first container or pod, all with only one click.
+
+![images empty page](img/podman-desktop-release-1.13/images_empty_page.png)
+
+### New Kubernetes navigation
+
+As we grow our Kubernetes section and add new Kubernetes related pages, the navigation was becoming a bit crowded. For better user exprience, this release includes a new Kubernetes sub menu with all the relevant pages in it.
```
Comment (shipsing) [inline website/blog/2024-10-07-release-1.13.md]:
```suggestion
As we grew our Kubernetes section and added new Kubernetes pages, the navigation was becoming a bit crowded. For a better user experience, this release includes a new Kubernetes submenu with all the relevant pages in it.
```

File: website/blog/2024-10-07-release-1.13.md [hunk 7]
```diff
@@ -0,0 +1,120 @@
...
+![podman-desktop-hero-1.13](img/podman-desktop-release-1.13/banner.png)
+
+Podman Desktop 1.13 is now available. [Click here to download it](/downloads)!
+
+This release includes:
+
+- **HyperV support**: You can now create and manage HyperV Podman machines directly from Podman Desktop.
+- **Search image feature**: Search for an image from Podman Desktop.
+- **Updated empty state pages**: Start your journey in the containers, images, pods, and Kubernetes pages with a click of a button
+- **New Kubernetes navigation**: All Kuberenetes related pages have been moved to a separate sub menu for easier and less crowded navigation.
```
Comment (Firewall) [inline website/blog/2024-10-07-release-1.13.md]: typo `Kuberenetes` -> `Kubernetes`

**Review by Firewall — CHANGES_REQUESTED** [review]

File: website/blog/2024-10-07-release-1.13.md (hunk 7 above)
Comment (axel7083) [reply website/blog/2024-10-07-release-1.13.md]: @Firewall you can probably use the `suggestion` option when there is a small change :)

```suggestion
- **New Kubernetes navigation**: All Kubernetes related pages have been moved to a separate sub menu for easier and less crowded navigation.
```

https://github.com/user-attachments/assets/bb9c9b46-163a-4d96-8536-c5607a039cf3

Comment (Firewall) [inline website/blog/2024-10-07-release-1.13.md]:
```suggestion
- **New Kubernetes navigation**: All Kubernetes related pages have been moved to a separate sub menu for easier and less crowded navigation.
```

File: website/blog/2024-10-07-release-1.13.md [hunk 8]
```diff
@@ -0,0 +1,120 @@
...
+- **New Kubernetes navigation**: All Kubernetes related pages have been moved to a separate sub menu for easier and less crowded navigation.
+
+<!--truncate-->
+
+## Release Details
+
+### HyperV support
+
+Podman Desktop is now supporting the creation and modification of HyperV based Podman machines in Windows directly from the app.
+When creating a Podman machine in Windows, if both HyperV and WSL are enabled, you can choose either one of them as the provider from the dropdown menu.
```
Comment (benoitf) [inline website/blog/2024-10-07-release-1.13.md]:
```suggestion
When creating a Podman machine in Windows, if both Hyper-V and WSL are enabled, you can choose either one of them as the provider from the dropdown menu.
```

File: website/blog/2024-10-07-release-1.13.md [hunk 9]
```diff
@@ -0,0 +1,120 @@
...
+
+Podman Desktop 1.13 Release! 🎉
+
+![podman-desktop-hero-1.13](img/podman-desktop-release-1.13/banner.png)
+
+Podman Desktop 1.13 is now available. [Click here to download it](/downloads)!
+
+This release includes:
+
+- **HyperV support**: You can now create and manage HyperV Podman machines directly from Podman Desktop.
```
Comment (benoitf) [inline website/blog/2024-10-07-release-1.13.md]:
```suggestion
- **Hyper-V support**: You can now create and manage Hyper-V Podman machines directly from Podman Desktop.
```

**Review by benoitf — COMMENTED** [review]
I don't see anything about the experimental flag for the docker compatibility mode and also maybe an highlight on a new extension available in the catalog layers explorer (once installed you can inspect the layers of an image)

**Review by benoitf — COMMENTED** [review]
I would also add something on the possibility to keep/adjust window size in the settings as it was something asked by a lot of users

- showcase the release-notes section in the dashboard

- remembering the position was also asked by  feedback so probably need to make a dedicated line/explanation rather than just a Other notable enhancements

File: website/blog/2024-10-07-release-1.13.md [hunk 10]
```diff
@@ -0,0 +1,140 @@
...
+
+### New Kubernetes navigation
+
+As we grew our Kubernetes section and added new Kubernetes related pages, the navigation was becoming a bit crowded. For a better user experience, this release includes a new Kubernetes submenu with all the relevant pages in it.
+
+![Kubernetes menu](img/podman-desktop-release-1.13/kubernetes_menu.png)
+
+### Image layer explorer extension
+
+You can now see each layer of any image you have in Podman Desktop using the `Image Layers Explorer` extension from the extensions catalog. After downloading the extension, choose any image from the available image list, go to the `Files` tab in the `Image Details` page, click on `Fetch Layers`, and start exploring.
```
Comment (SoniaSandler) [inline website/blog/2024-10-07-release-1.13.md:56]:
```suggestion
You can now inspect each layer of any image you have in Podman Desktop using the `Image Layers Explorer` extension from the extensions catalog. After downloading the extension, choose any image from the available image list, go to the `Files` tab in the `Image Details` page, click on `Fetch Layers`, and start exploring.
```

**Review by benoitf — APPROVED** [review]
thanks @SoniaSandler

**Comment by benoitf** [comment]
@Firewall if you can review again as it seems your remarks have been fixed

**Review by Firewall — APPROVED** [review]

File: website/blog/2024-10-07-release-1.13.md [hunk 11]
```diff
@@ -0,0 +1,140 @@
...
+![podman-desktop-hero-1.13](img/podman-desktop-release-1.13/banner.png)
+
+Podman Desktop 1.13 is now available. [Click here to download it](/downloads)!
+
+This release includes:
+
+- **Hyper-V support**: You can now create and manage Hyper-V Podman machines directly from Podman Desktop.
+- **Search image feature**: Search for an image from Podman Desktop.
+- **Updated empty state pages**: Start your journey in the containers, images, pods, and Kubernetes pages with a click of a button
+- **New Kubernetes navigation**: All Kuberenetes related pages have been moved to a separate sub menu for easier and less crowded navigation.
```
Comment (jeffmaury) [inline website/blog/2024-10-07-release-1.13.md]:
```suggestion
- **New Kubernetes navigation**: All Kubernetes related pages have been moved to a separate sub menu for easier and less crowded navigation.
```

File: website/blog/2024-10-07-release-1.13.md [hunk 12]
```diff
@@ -0,0 +1,140 @@
...
+- **Image Layer explorer extension**: With this new extension, you can explore the various layers of images.
+- **Experimental Docker compatibility mode page**: You can now try out our experimental docker compatibility mode page.
```
Comment (jeffmaury) [inline website/blog/2024-10-07-release-1.13.md]:
```suggestion
- **Experimental Docker compatibility mode page**: You can now try out our experimental Docker compatibility mode page.
```

File: website/blog/2024-10-07-release-1.13.md [hunk 13]
```diff
@@ -0,0 +1,140 @@
...
+<!--truncate-->
+
+## Release Details
+
+### Hyper-V support
+
+Podman Desktop is now supporting the creation and modification of Hyper-V based Podman machines in Windows directly from the app.
```
Comment (jeffmaury) [inline website/blog/2024-10-07-release-1.13.md]: Not sure between in Windows or on Windows is better ?

Comment (shipsing) [reply website/blog/2024-10-07-release-1.13.md]: on Windows

**Pull request merged**